client/js/src/main/scala/zio/zmx/client/frontend/state/WebsocketHandler.scala

```diff
@@ -26,18 +29,22 @@ object WebsocketHandler {
 
   def sendCommand(msg: ClientMessage): Unit = {
     val url   = AppState.connectUrl.now()
-    val wsCmd = WebSocket.url(url).arraybuffer.build(managed = false, autoReconnect = false)
-    val f     = wsFrame(msg)
+    val wsCmd = newArrayBufferWebSocket(url).build(
+      managed = false,
+      autoReconnect = false
+    )
+
     println(s"Sending WS message <$msg> to <$url>")
-    try {
-      wsCmd.reconnectNow()
-      wsCmd.sendOne(f)
-    } catch {
+    try wsCmd.reconnect
+      .onNext(
+        wsCmd.sendOne(wsFrame(msg))
+      )
```

Comment:
This got rid of a warning in the JS console, basically sequencing `reconnect` and `sendOne`.